src/cards/ResourceCard/style.js

```diff
@@ -0,0 +1,18 @@
+import { makeStyles } from '@material-ui/core';
```

Comment:
I'm not sure we need this css file because mui components have already the layout we need, the only thing missing is `fullWidth={true}` in the Button component of `<CardActions />` :slightly_smiling_face: